backend/src/main/java/ai/verta/modeldb/project/ProjectServiceImpl.java

```diff
@@ -1330,47 +1277,6 @@ public void getProjectShortName(
     }
   }
 
-  @Override
-  public void setProjectVisibility(
-      SetProjectVisibilty request, StreamObserver<SetProjectVisibilty.Response> responseObserver) {
-    try {
-      // Request Parameter Validation
-      if (request.getId().isEmpty()) {
-        String errorMessage = "Project ID not found in SetProjectVisibilty request";
-        LOGGER.info(errorMessage);
-        Status status =
-            Status.newBuilder()
-                .setCode(Code.INVALID_ARGUMENT_VALUE)
-                .setMessage(errorMessage)
-                .addDetails(Any.pack(SetProjectVisibilty.getDefaultInstance()))
-                .build();
-        throw StatusProto.toStatusRuntimeException(status);
-      }
-
-      // Validate if current user has access to the entity or not
-      roleService.validateEntityUserWithUserInfo(
-          ModelDBServiceResourceTypes.PROJECT, request.getId(), ModelDBServiceActions.UPDATE);
-
-      Project project =
-          projectDAO.setProjectVisibility(request.getId(), request.getProjectVisibility());
-      saveAuditLogs(
-          null,
-          ModelDBConstants.UPDATE,
-          Collections.singletonList(project.getId()),
-          String.format(
-              ModelDBConstants.METADATA_JSON_TEMPLATE,
-              "update",
-              "visibility",
-              project.getProjectVisibility().name()));
-      responseObserver.onNext(
-          SetProjectVisibilty.Response.newBuilder().setProject(project).build());
-      responseObserver.onCompleted();
-    } catch (Exception e) {
-      ModelDBUtils.observeError(
-          responseObserver, e, SetProjectVisibilty.Response.getDefaultInstance());
-    }
-  }
-
```

Comment:
Why this endpoint is removed?? is it intentionally removed?? cc @conradoverta, @coryverta

Reply:
Ok, i found the Proto change for this https://github.com/VertaAI/protos-all/pull/99/files?file-filters%5B%5D=.proto#diff-05d06fb0d9c3d4b7b133e720c399f054fcea157e127a72581ca7d35cb86949e5L516-L522 so assuming it removed by intention.